package.json

```diff
@@ -37,6 +37,11 @@
           "default": "",
           "description": "The file where the navigation into the wiki starts (basically the wiki entry file)."
         },
+        "wiki.baseRootOpenAsFolder": {
+          "type": "boolean",
+          "default": "",
+          "description": "Open the entire folder containing the base file as a workspace."
+        },
```

Comment:
type is `boolean` but default value is empty string.

Can be set to false, the default value?

Reply:
Sure, good point. Didn't recognize this in the hurry.